src/frontends/tensorflow/src/translate_session.cpp

```diff
@@ -56,6 +56,14 @@ static bool apply_saved_model_names(std::shared_ptr<ov::Node> node,
     }
     return false;
 }
+
+void decode_input_port(std::shared_ptr<DecoderBase> node,
+                       size_t input_port_idx,
+                       std::string& producer_name,
+                       std::string& producer_output_port_name,
+                       size_t& producer_output_port_index) {
+    node->get_input_node(input_port_idx, producer_name, producer_output_port_name, producer_output_port_index);
```

Comment:
quite strange helper, also node != decoder:)
Propose to remove it.

Reply:
Yeah, it was obsolete